libs/ledger-live-common/src/hw/customLockScreenLoad.ts

```diff
@@ -213,6 +212,14 @@ export default function loadImage({ deviceId, request }: Input): Observable<Load
           sub.unsubscribe();
         };
       }),
+  ).pipe(
```

Comment:
[ask] isn't it the purpose of the optional argument `openTimeoutMs` of `withDevice` ?